release_dashboard/lib/state/status_state.dart

```diff
@@ -30,6 +30,26 @@ class StatusState extends ChangeNotifier {
 
     notifyListeners();
   }
+
+  /// Updates the release status with the latest values saved in the state file.
+  ///
+  /// Use the code below to call the method:
+  ///
+  /// {@tool snippet}
+  ///
+  /// [context] is the [BuildContext] of the widget which is calling this method.
```

Comment:
These parameters should be before the `Use the code below to call the method` line